sit-awareness/cv.py

```diff
@@ -0,0 +1,267 @@
+import math
+import cv2
+import numpy as np
+import subprocess
+import imghdr
+import traceback
+import os
+from math import sin, cos
+from operator import add
+
+# finds rotation and translation of vision targets
+class PoseDetector:
+
+	# initilaze variables
+	def __init__(self, input, target):
+
+		self.input_path = input
+
+		try:
+			# if input is a camera port
+			self.input = cv2.VideoCapture(int(input))
+			self.set_camera_settings(input)
+		except:
+			# if input is a path
+			self.input = cv2.VideoCapture(input)
+
+		self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 640)
+		self.input.set(cv2.CAP_PROP_FRAME_HEIGHT, 480)
+		frame = self.get_frame()
+
+		self.previous_r = []
+		self.previous_t = []
+
+		self.object_points = []
+
+		if target == "lb":
```

Comment:
I think we can afford to spell out loading bay and power port. Also, consider saving these arrays of points as class constants, and just have caller pass in the  array in the constructor.